apache/fineract-cn-mobile
# Added UI/Integration tests for creating Accounts, Accounts list, Account details and Accounts Task mgmt. features

This PR has been made on top of this [PR](https://github.com/apache/fineract-cn-mobile/pull/191)

Please Add Screenshots If any UI changes:

1. Test Account creation successfully 
- Enter Account identifier
- Enter name
- Enter Alternate account number
- Enter Ledger
- Enter Reference account number 
- Enter Balance 
- Navigate to the next stepper screen 
- Create a new Account holder 
- Edit existing Account holder 
- Try deleting the account holder 
- Navigate to the next screen 
- Create a new Signature authority 
- Navigate to the the final Review fragment of the stepper layout
- Check if the account was created successfully?
- delete the test entry from the database after the test is successful


https://user-images.githubusercontent.com/53621853/130322675-5729098e-c185-4769-a201-44bef45e5c4a.mp4


2. Test account state changed successfully
- launch the account list fragment 
- open the account details fragment 
- Click on the task linear layout 
- enter comment while changing the state
- Assert if the state was updated successfully?



https://user-images.githubusercontent.com/53621853/130322694-3723c2e6-95b1-435a-b9db-877071d5f54b.mp4



3. Test Account List fragment and recyclerView onclick
- Launch the application 
- Navigate to the teller list fragment and check if recyclerview is visible
- click on the recyclerView Item
- Assert if the item has been displayed correctly


Please Add Summary of what changes you have made and if possible provide links.(Optional)

Please make sure these boxes are checked before submitting your pull request - thanks!


- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.
